### ISSUE TITLE
Cannot declare class CreateWebauthnCredentialsTable, because the name is already in use

### ISSUE THREAD

**Issue by luiyongsheng** [m1guelpf/laravel-fastlogin#10]
Error is thrown when I do migrations for the first time.

**Comment by niladam** [comment]
@luiyongsheng How is that helpful ? Can you please try and clarify ? Maybe screenshots ? Maybe pieces of code ?

**Comment by luiyongsheng** [comment]
![image](https://user-images.githubusercontent.com/13586871/124143788-fdffb580-dabd-11eb-9d2c-3b68fafc80e1.png)

**Comment by luiyongsheng** [comment]
The only code change after `m1guelpf/laravel-fastlogin` package installed
```PHP
class User extends Authenticatable
{
    // ....
    use CanFastLogin;
    // ....
}
```

**Comment by philipmclifton** [comment]
Same issue nothing more than composer install and added the use

**Comment by niladam** [comment]
It appears you have the same class. Maybe you published it twice?

**Comment by philipmclifton** [comment]
Only commands run:

 composer require m1guelpf/laravel-fastlogin

php artisan migrate

**Comment by luiyongsheng** [comment]
> Only commands run:
> 
> 
> 
>  composer require m1guelpf/laravel-fastlogin
> 
> 
> 
> php artisan migrate

same here

**Comment by FatlumGjinofci** [comment]
same error for me

**Comment by FatlumGjinofci** [comment]
same error for me

**Comment by philipmclifton** [comment]
v1.0.2 does not have this issue.

**Comment by FatlumGjinofci** [comment]
> v1.0.2 does not have this issue.

yep v1.0.2 worked fine for me

**Comment by dirk-helbert** [comment]
You have to rename the migration file

from 2021_01_24_154837_create_credentials_table.php

to 2021_01_24_154837_create_webauthn_credentials_table.php

**Comment by luiyongsheng** [comment]
I think the README file should get an update, instructions are not clear.

**Comment by philipmclifton** [comment]
> You have to rename the migration file
> 
> from 2021_01_24_154837_create_credentials_table.php
> 
> to 2021_01_24_154837_create_webauthn_credentials_table.php
> 
> Pull request is created
> 
> #11

Confirmed - cheers!

**Comment by Muskulee** [comment]
> > You have to rename the migration file
> > from 2021_01_24_154837_create_credentials_table.php
> > to 2021_01_24_154837_create_webauthn_credentials_table.php
> > Pull request is created
> > #11
> 
> Confirmed - cheers!
Makes sense

**Comment by philipmclifton** [comment]
Any chance we can get this merged in please?

**Comment by m1guelpf** [comment]
Fixed in #11
